library/src/main/java/cc/mvdan/accesspoint/WifiApControl.java

```diff
@@ -436,4 +437,57 @@ public void run() {
 		}.start();
 		return clients;
 	}
+
+	/**
+	 * Checks if 5Ghz band is available to create an accesspoint.
+	 * @return
+	 */
```

Comment:
Use shorter method docs, like in the rest of the code:

```
// Checks if 5Ghz band is available to create an accesspoint.
```

Same for the rest of the code you added.